src/frontend/planner_test/tests/testdata/expr.yaml

```diff
@@ -351,3 +351,22 @@
   batch_plan: |
     BatchProject { exprs: ['integer[][]':Varchar] }
     └─BatchValues { rows: [[]] }
+- name: now expression
+  sql: |
+    create table t (v1 timestamp);
+    select * from t where v1 >= now();
+  logical_plan: |
+    LogicalProject { exprs: [t.v1] }
+    └─LogicalFilter { predicate: (t.v1 >= Now) }
+      └─LogicalScan { table: t, columns: [t.v1, t._row_id] }
+  stream_plan: |
+    StreamMaterialize { columns: [v1, t._row_id(hidden)], pk_columns: [t._row_id] }
+    └─StreamDynamicFilter { predicate: (t.v1 >= t.v1), output: [t.v1, t._row_id] }
+      ├─StreamFilter { predicate: true }
+      | └─StreamTableScan { table: t, columns: [t.v1, t._row_id], pk: [t._row_id], dist: UpstreamHashShard(t._row_id) }
+      └─StreamNow { output: [now] }
+- name: invalid now expression
+  sql: |
+    create table t (v1 timestamp, v2 timestamp);
+    select * from t where v1 >= now() or v2 >= now();
+  stream_error: 'Expr error: Invalid parameter now: now expression must be placed in a comparison'
```

Comment:
Could we try with `select * from t where v1 >= now() and v2 >= now();` too? I think this should work (two dynamic filters).

Reply:
> Could we try with `select * from t where v1 >= now() and v2 >= now();` too? I think this should work (two dynamic filters).

fixed